code/obj/item/gun/kinetic.dm

```diff
@@ -870,6 +880,267 @@ ABSTRACT_TYPE(/obj/item/survival_rifle_barrel)
 			projectiles = list(current_projectile)
 			UpdateIcon()
 
+/obj/item/gun/kinetic/uzi
+	desc = "A stamped metal PDW."
+	name = "\improper MOR-30"
+	icon_state = "uzi"
+	item_state = "uzi"
+	spread_angle = 8
+	shoot_delay = 5
+	has_empty_state = TRUE
+	w_class = W_CLASS_SMALL
+	force = MELEE_DMG_PISTOL
+	ammo_cats = list(AMMO_PISTOL_9MM_ALL)
+	max_ammo_capacity = 30
+	auto_eject = TRUE
+	fire_animation = TRUE
+	default_magazine = /obj/item/ammo/bullets/nine_mm_surplus/mag_mor
+
+	get_desc(dist, mob/user)
+		if (user.get_gang() != null)
+			. += "For when you need MOR' DAKKA. Uses 9mm NATO rounds."
+		else
+			. += "Its' firemodes are labelled 'DAKKA' and 'MOR'... Uses 9mm NATO rounds."
+
+	New()
+		ammo = new default_magazine
+
+		set_current_projectile(new/datum/projectile/bullet/nine_mm_surplus/burst)
+		projectiles = list(current_projectile, new/datum/projectile/bullet/nine_mm_surplus/auto)
+		AddComponent(/datum/component/holdertargeting/fullauto, 1.5, 1.5, 1)
+		..()
+
+	attack_self(mob/user)
+		..()	//burst shot has a slight spread.
+		if (istype(current_projectile, /datum/projectile/bullet/nine_mm_surplus/auto))
+			spread_angle = 10
+			shoot_delay = 4
+		else
+			spread_angle = 8
+			shoot_delay = 5
+
+
+/obj/item/gun/kinetic/greasegun
+	name = "\improper Grease Gun"
+	desc = "A stamped metal SMG, cheaply produced years ago, and found in the weirdest places today."
+	icon_state = "grease"
+	item_state = "grease"
+	icon = 'icons/obj/items/guns/kinetic48x32.dmi'
+	spread_angle = 8
```

Comment:
I feel like the spread angle could be a little worse when not two-handing to really encourage it? The difference is kind of minor at non off-the-screen ranges